### DISCUSSION TITLE
New Updater Script

### DISCUSSION THREAD

**Discussion by J4k-The-Reaper** [UltimaHoarder/UltimaScraper#1356] [General]
its awesome you created the updater. i do use the Github Desktop app.....wondering if i still need to use the script or does the app update it automatically when i pull/refresh latest commits within it?

**Comment by UltimaHoarder** [comment]
Not sure, you'd have to test it. The script gets overwritten and using updater.py

**Reply by J4k-The-Reaper** [reply]
Okay. How would I know it worked or not? Is there a way to see in script or something if it got updated?

**Reply by UltimaHoarder** [reply]
It will work or throw an error

**Comment by DannyTwice** [comment]
after i downloaded the most recent update, python doesnt even run the code, it just closes down. dont understand why

**Reply by Throwaway115** [reply]
having the same.
It said it apperently updated the very first time i ran it but it just made the folders, now just closes when i try run it

**Reply by UltimaHoarder** [reply]
No one opened an issue for it. Anyway, I've fixed it.

**Reply by Throwaway115** [reply]
> No one opened an issue for it. Anyway, I've fixed it.

Thank you!
